modules/ROOT/pages/backup-and-disaster-recovery.adoc

```diff
@@ -7,6 +7,8 @@ To preserve your data in case of system failure or problems with other operation
 
 Anypoint Platform Private Cloud Edition (Anypoint Platform PCE) requires you to install, configure, and manage an NFS server. Ensure that you configure and perform frequent backups of this server.
 
+Anypoint Monitoring and Visualizer backup and restore are outside the normal Gravity backup and restore mechanism, and require a separate procedure.
```

Comment:
This should be highlighted or something as an important note

Reply:
Revised wording and added note.